draft-ietf-quic-transport.md

```diff
@@ -5610,7 +5619,7 @@ IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
 The Retire Prior To field counts connection IDs established during connection
-setup and the preferred_address transport parameter (see {{retiring-cids}}). The
+setup and the preferred_address transport parameter (see {{retire-cid}}). The
```

Comment:
```suggestion
setup and the preferred_address transport parameter; see {{retire-cid}}. The
```

Reply:
I thought parens were fine?  Personally, I prefer parens, so I'm curious why prefer semicolons?

I think they're used quite widely, so if you want to consistently move to semicolons, I'd suggest a single cleanup PR to switch everything over.